services/workflows-service/src/env.ts

```diff
@@ -78,6 +78,13 @@ export const env = createEnv({
       .string()
       .optional()
       .describe('Bucket name of Data migration folders'),
+    TELEMETRY_ENABLED: z
+      .enum(['true', 'false'])
+      .default('true')
+      .transform(value => value === 'true')
+      .describe('Enable or disable telemetry'),
+    TELEMETRY_SUPABASE_URL: z.string().url().optional().describe('Supabase URL for telemetry'),
```

Comment:
Rather than using a a DB. 
How about we have another webservice like analytics.ballerine.com 
to which we make an api call and track user activity ? more like